CMakeLists.txt

```diff
@@ -20,7 +20,7 @@ cmake_minimum_required(VERSION 3.10)
 option(BUILD_SHARED_LIBS "Build using shared libraries" ON)
 option(BUILD_DOCS "Enable autodoc building" OFF)
 option(BUILD_AGENT "Build the Agent library and executable" ON)
-option(BUILD_MANAGER "Build the Manager library and executable" ON)
+option(BUILD_MANAGER "Build the Manager library and executable" OFF) # FIXME default to ON
```

Comment:
**Question**: Do we want to create an issue to track this or is a FIXME good enough?

Reply:
There will be several tickets after the apl-fy24 merge, updating the manager will be one.

Reply:
This is captured in #68